_includes/_head.html

```diff
@@ -14,4 +14,17 @@
 
   <link rel="stylesheet" href="{{ site.cssurl | relative_url }}" />
     <!-- Global site tag (gtag.js) - Google Analytics -->
+
+  <!-- Tealium Tag Manager - SYNC -->
+  <script src="https://tags.tiqcdn.com/utag/cmsgov/cms-dpc/prod/utag.sync.js"></script>
+
+  <!-- Tealium Tag Manager - ASYNC -->
+  <script type="text/javascript">
+    (function(a,b,c,d){
+    a='https://tags.tiqcdn.com/utag/cmsgov/cms-dpc/prod/utag.js';
```

Comment:
I do think we want to set the `utag_data` global var / separate synchronous script before we load this, as mentioned in the confluence instructions and in the email from Josh. 

It looks like there's a utag_data already set up for BCDA, although it's still from the Google Analytics days so the actual `utag_data` keys are different.

https://github.com/CMSgov/bcda-static-site/blob/7795a67343880880f23faa374bfe3902b235bd25/_layouts/default.html#L8

For us it should look a bit like this:

```js
var utag_data = {
  "content_language": "en",
  "page_name": "{{ page.title }}",
  "page_path": "{{ page.url }}",
  "site_domain": "https://dpc.cms.gov",  // or "https://stage.dpc.cms.gov"
  "site_environment": "{{ jekyll.environment }}",
}
```

When I follow the instructions in the Confluence page to look in the console, there's a noticeable difference in the events with and without this set up.

|With|Without utag_data|
|---|---|
|<img width="1283" alt="Screen Shot 2023-10-11 at 5 57 57 PM" src="https://github.com/CMSgov/dpc-static-site/assets/2308368/eea181f0-9f8e-4510-9230-fab2e442651e">|<img width="1185" alt="Screen Shot 2023-10-11 at 6 01 26 PM" src="https://github.com/CMSgov/dpc-static-site/assets/2308368/7f2ea8c9-97fa-44c7-9392-76c073d91010">|

There are more events emitted after the pre-init step that all contain the `utag_data` when it's set.

Reply:
ah, thanks, I was a little confused about where the body template was